config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -117,8 +117,8 @@ presubmits:
         - -c
         - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
         env:
-        - name: FOCUS
-          value: "."
+        - name: LABEL_FILTER
+          value: "!/Feature:.+/"
         # TODO(bentheelder): reduce the skip list further
         - name: SKIP
           value: \[Slow\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[sig-storage\]|PodSecurityPolicy|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing
```

Comment:
why skip sig-storage ?

Reply:
those tests used to be very flaky

Reply:
This SKIP is historic. It looks like some tests were specifically added probably because they didn't work in this environment, others (like this blanket exclusion of sig-storage) perhaps because they don't add value and ran too long.

As @BenTheElder said above in the TODO, this list should get reduced. I'm just not sure how to do that safely. Aggressively prune, monitor the result, and fix it up afterwards if something becomes worse (test failures, significantly longer runtime)?

Unrelated to that, when using `LABEL_FILTER` let's drop the `SKIP`. That makes the jobs simpler.

Reply:
let's do incremental steps, if we do multiple changes at the same time and something fails we'll not be able to trace it back easily

Reply:
Then at least remove `|\[Feature:.+\]` from `SKIP` because that is what `"!/Feature:.+/"` replaces. Otherwise it's not different from before.

Actually, let's not use the regexp. Instead: `Feature: isEmpty`.

Reply:
@pohly updated.

Reply:
There's a "canary" job for iterating on changes before promoting them to the always-run job

Reply:
So should we do one trial with that one before merging this PR? The PR looks good to me, but getting it wrong would be disruptive.

@carlory: can you convert one such "canary" job?

Reply:
I would recommend it, you can PR the canary, test it on a Kubernetes PR with no real changes.

Reply:
@BenTheElder @pohly please review https://github.com/kubernetes/test-infra/pull/32898

Reply:
I see a different number of test results:
- 918 for canary: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/125163/pull-kubernetes-e2e-kind-canary/1808785648497201152
- 895 for normal job (checked a few, here's one): https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/125877/pull-kubernetes-e2e-kind/1808746186597732352

@carlory: can you check where that difference comes from?

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/125163/pull-kubernetes-e2e-kind/1808799504367882240 will be for the exact same PR where I also ran the canary job, to rule out changes because of the PR.

Reply:
> @carlory: can you check where that difference comes from?

@pohly  normal job skips more items than the canary.

```
|NFS|nfs|inline.execution.and.attach|should.be.rejected.when.no.endpoints.exist
```

Reply:
I don't know why to skip those tests

Reply:
those were flaky in the past , or didn't work for any other reason, ... I can't remember honestly

Reply:
should we remove those skip from normal jobs?  the canary job is successful. @aojea

Reply:
We need to find a balance between periodic and presubmits, I don't think we should run the nfs tests on presubmit, periodics are ok for that

> inline.execution.and.attach

this was very flaky I don't know if has been fixed now

> should.be.rejected.when.no.endpoints.exist

this should be ok to run now IIRC

Reply:
Whatever we do with these tests specifically, should we keep the canary job identical to the normal job?

Reply:
if canary show stability and reasonable duration we can reduce the skip list here

Reply:
@pohly @aojea I open a PR to make the canary job identical to the normal job.  https://github.com/kubernetes/test-infra/pull/32907

Let’s reduce the SKIP list later. 

I want to make ci pass in  https://github.com/kubernetes/kubernetes/pull/125452

Reply:
kindly ping @pohly

Reply:
I'm okay with keeping the SKIP different. Perhaps add a comment about it to the YAML?

Reply:
Also cross-reference the jobs and explain that one should update the canary first when making potentially intrusive changes. This is tribal knowledge - we should start documenting this.

Reply:
@pohly added. Please review it again. thanks